src/Tizen.Applications.ComponentBased.ComponentPort/Tizen.Applications.ComponentBased.ComponentPort/ComponentPort.cs

```diff
@@ -89,6 +88,63 @@ public void AddPrivilege(string privilege)
         /// <summary>
         /// Waits for events.
         /// </summary>
+        /// <remarks>
+        /// This method runs a main loop until Cancel() is called.
+        /// The code in the next line will not run until Cancel() is called.
+        /// It is recommended that this method uses in the sub thread.
+        /// </remarks>
+        /// <example>
+        /// <code>
```

Comment:
What is the point of this `<code>` section? First of all, it is not related to this method. Second of all, it is too long and redundant.

Reply:
I've fixed it.